### ISSUE TITLE
runMonkeyPatchForGoogleTranslate is polluting bugsnag

### ISSUE THREAD

**Issue by ikesau** [owid/owid-grapher#1785]
## Description

The `runMonkeyPatchForGoogleTranslate` function  in [site/hacks.ts](https://github.com/owid/owid-grapher/blob/master/site/hacks.ts#L11) is throwing errors that get reported to Bugsnag thousands of times a month.

Both `"Cannot insert before a reference node from a different parent"` and `"removeChild: Cannot remove a child from a different parent"`

Cursory attempts by @marcelgerber and I to reproduce this error haven't worked, but given how much noise it adds to our slack channel, I think it would be worth a day to try and resolve.

Ideally when people use Google translate on our pages, it should translate the whole page without any errors.

If it's able to translate the page but the function still trips up because of React re-rendering or something, then I'd suggest not logging these errors or finding a way to permanently ignore them in Bugsnag.

**Comment by marcelgerber** [comment]
Some important context on what use cases in React are affected, and how one can rewrite them, [can be found in this very good comment (from 2018!) over in the React repo](https://github.com/facebook/react/issues/11538#issuecomment-390386520).

---

In good news, I was able to repro the issue locally. I set up a branch that makes it relatively easy to repro hopefully:
1. Checkout https://github.com/owid/owid-grapher/tree/google-translate-issue-repro and run from there
2. Go to http://localhost:3030/grapher/share-of-cumulative-co2, which contains a DoD
3. In the 5 seconds between page load and the delayed details load, translate the page to some language
4. => You will get an error message after the 5 second delay is up and the DoD load

On live, I can relatively easily repro the issue on https://ourworldindata.org/co2/country/argentina by translating it and then scrolling some.

---

It seems that converting some `<React.Fragment>` to `<span>`s in `MarkdownTextWrap` does the trick (in particular, in IRText, IRWhitespace). Currently, this makes Google Translate delete spaces between words though, so that's also not ideal.
But will have to try this out some more. We can also pair on this at some point.
